docker/php/Dockerfile

```diff
@@ -0,0 +1,9 @@
+FROM php:7.1-fpm
+
+RUN apt-get update
+
+RUN apt-get install git unzip -y
+RUN curl -sS https://getcomposer.org/installer | php -- --install-dir=/usr/local/bin --filename=composer
+RUN composer --version
```

Comment:
Is this line needed for something? Can we remove it?

Reply:
Yes, we can remove it.